src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -360,7 +360,7 @@ class MetaInfoPanelImpl extends React.PureComponent<Props, State> {
           {meta.extensions
             ? _renderRowOfList(
                 'MenuButtons--metaInfo-renderRowOfList-label-extensions',
-                meta.extensions.name
+                Array.from(meta.extensions.name).sort() // copy the array because sort is in-place.
```

Comment:
Interesting use of Array.from, usually we use .slice() for these things, but I don't mind :-)

Reply:
Is this because `Array.slice` is available on older browsers? As someone who doesn't closely follow JavaScript, `Array.from` seems like an intuitive copy method and I wouldn't have thought about `Array.slice` 😆